src/screenComponents/shipsLogInternControl.cpp

```diff
@@ -0,0 +1,85 @@
+#include "playerInfo.h"
+#include "spaceObjects/playerSpaceship.h"
+#include "shipsLogInternControl.h"
+
+#include "gui/gui2_panel.h"
+#include "gui/gui2_advancedscrolltext.h"
+
+ShipsLogIntern::ShipsLogIntern(GuiContainer* owner)
```

Comment:
This looks like a copy&paste from the normal ships log. It's better to have a single class that can serve both, instead of copy&pasting.